server/base/src/main/java/org/apache/accumulo/server/constraints/MetadataConstraints.java

```diff
@@ -211,9 +214,9 @@ public List<Short> check(Environment env, Mutation mutation) {
         } catch (NumberFormatException | ArrayIndexOutOfBoundsException nfe) {
           violations = addViolation(violations, 1);
         }
-      } else if (columnFamily.equals(ScanFileColumnFamily.NAME)) {
-
-      } else if (columnFamily.equals(BulkFileColumnFamily.NAME)) {
+      } else if (columnFamily.equals(ScanFileColumnFamily.NAME)) {} else if (columnFamily
```

Comment:
Is there a future reason to have two separate `else if (x) {} else if (y) {}` no-op statements vs a single `else if (x or y)`? 

If so, there are now four distinct comparisons on `columnFamily.equals` for `TEXT` values. Could this be simplified into a switch statement for readability?

Reply:
> Is there a future reason to have two separate else if (x) {} else if (y) {} no-op statements vs a single else if (x or y)?

No reason, just keeping my change in line with the others. We can certainly collapse them before this is merged.